teps/0121-refine-retries-for-taskruns-and-customruns.md

```diff
@@ -78,6 +79,201 @@ They set both `retries` and `timeout` for the two resources, under the current i
 - `task-run-example` will be retried once after 10s.
 - `custom-run-example` will be timed out after 10s. But if the corresponding CustomRun controller implements retries **for *each* attempt**, like in TaskRuns, instead of **for all attempts** per the documented guidance, then the `custom-run-example` would be retried once after 10s, working similarly to the `task-run-example`.
 
+#### Retries in Custom Runs
+
+Some Custom Tasks need retry logic that is different from just creating a new instance of a Run when a previous attempt has failed, as described in [TEP-0069](./0069-support-retries-for-custom-task-in-a-pipeline.md).
+One example is the [TaskLoop custom task](https://github.com/tektoncd/experimental/tree/main/task-loops), which creates one TaskRun for each value in an array parameter for a single Task.
+If retries are specified, only failed TaskRuns are retried. Creating a new Run would retry every TaskRun, even the successful ones.
+
+```yaml
+apiVersion: custom.tekton.dev/v1alpha1
+kind: TaskLoop
+metadata:
+  name: testloop
+spec:
+  taskRef:
+    name: my-task
+  iterateParam: test-type
+  retries: 2
```

Comment:
Since `Run` already provides a `retries` field, having `retries` defined for `TaskLoop` seems unlikely to me.